io_u.c

```diff
@@ -895,7 +915,13 @@ static int fill_io_u(struct thread_data *td, struct io_u *io_u)
 		goto out;
 
 	set_rw_ddir(td, io_u);
-
+	/*
+	 * check if we have received Invalid Direction
+	 */
```

Comment:
If we're keeping the comment keep it all lower case to match the style of the other comments. I'd argue you don't need this particular comment as it is basically saying what the condition of the if is saying (contrast with the comment below talking about fsync where's not immediately obvious the if is dealing with fsync...

Reply:
I will remove this comment